lib/common_fw/common_fw.cpp

```diff
@@ -32,40 +34,93 @@ bool write_to_SD(std::string message, const char * file_name) {
     return true;
 }
 
+
+/**
+ *
+ */
+bool write_to_SD(std::string message, const char * file_name) {
+    return write_to_SD((uint8_t *)message.c_str(), message.length(), file_name);
+}
```

Comment:
When is this function used? Why don't we just use the one above?

Reply:
Debug. When we're writing constant strings see`setup`